peachjam_search/views.py

```diff
@@ -318,6 +318,45 @@ def __init__(self, *args, **kwargs):
         self.index = get_search_indexes(self.document._index._name)
         self.search = self.search.index(self.index)
 
+    def get_translatable_fields(self, request):
+        # get language from request
+        language = request.LANGUAGE_CODE
+        # suffix for translatable fields e.g. court_en, court_fr etc
+        suffix = ""
+        if language in [lang for lang, _ in settings.LANGUAGES]:
+            suffix = "_" + language
+
+        translatable_facets = {
+            "court": {"field": f"court{suffix}", "options": {"size": 100}},
```

Comment:
The configuration is now duplicated.

Instead, adjust the existing configuration -- only the `field` field needs to change.

Reply:
Perhaps a hard-coded list of the translatable fields (eg. `["court", "nature", ...]`) which you then use to introspect and adjust the `facets` and `filters` fields.